test/adapter/json/polymorphic_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    module Adapter
+      class Json
+        class PolymorphicTest < ActiveSupport::TestCase
+          setup do
+            @employee = Employee.new(id: 42, name: 'Zoop Zoopler', email: 'zoop@example.com')
+            @picture = @employee.pictures.new(id: 1, title: 'headshot-1.jpg')
+            @picture.imageable = @employee
+
+            @serializer = PictureSerializer.new(@picture)
+            @adapter = ActiveModel::Serializer::Adapter::Json.new(@serializer)
```

Comment:
Yeah, sorry for late comments... This is great pr and thanks for work... Just saw with fresh eyes, I guess... On phone todayy then traveking tomorow sonwanted to take a look